lib/osml/data_catalog/dc_stac_container.ts

```diff
@@ -0,0 +1,122 @@
+/*
+ * Copyright 2024 Amazon.com, Inc. or its affiliates.
+ */
+
+import { RemovalPolicy, SymlinkFollowMode } from "aws-cdk-lib";
+import { DockerImageCode } from "aws-cdk-lib/aws-lambda";
+import { Construct } from "constructs";
+
+import { OSMLAccount } from "../osml_account";
+import { OSMLECRDeployment } from "../osml_ecr_deployment";
+import { OSMLVpc } from "../osml_vpc";
+
+/**
+ * Represents the configuration for the DCStacContainer Construct.
+ */
+export class DCStacContainerConfig {
+  /**
+   * Creates an instance of DCStacContainerConfig.
+   * @param {string} [DC_STAC_CONTAINER="awsosml/awsosml/osml-data-intake-stac:latest"] - Container image pull.
+   * @param {string} [DC_STAC_BUILD_PATH="lib/osml-data-intake"] - The build path for the source code.
+   * @param {string} [DC_STAC_BUILD_TARGET="osml_data_intake_stac"] - The build target within the Dockerfile.
+   * @param {string} [DC_STAC_REPOSITORY="osml-data-intake-stac-container"] - The name to give the ECR repo.
+   * @param {string} [DC_STAC_DOCKERFILE="docker/Dockerfile.stac"] - The dockerfile to build with.
+   */
+  constructor(
+    public DC_STAC_BUILD_PATH: string = "lib/osml-data-intake/",
+    public DC_STAC_CONTAINER: string = "awsosml/osml-data-intake-stac:latest",
+    public DC_STAC_BUILD_TARGET: string = "stac",
+    public DC_STAC_REPOSITORY: string = "data-intake-stac-container",
+    public DC_STAC_DOCKERFILE: string = "docker/Dockerfile.stac"
+  ) {}
+}
+
+/**
+ * Interface representing the properties for the DCFastAPIContainer construct.
```

Comment:
Is it DCFastAPIContainer or DCStacContainer

Reply:
StacContainer